linter/rules/yesno_field_label_contains_yesno.py

```diff
@@ -0,0 +1,13 @@
+from linter.rule import Rule
+from typing import Any, Tuple
+from re import search, IGNORECASE
+
+
+class YesnoFieldLabelContainsYesno(Rule):
+    def applies_to() -> Tuple[str, ...]:
+        return ('dimension', 'measure')
+
+    def run(self, field: Any) -> bool:
+        if field.get('type') == 'yesno':
```

Comment:
I'm surprised anyone would be adding yes/no to a label, that seems so arbitrary.  And it's not the exact thing Looker adds either (which is "(Yes / No)".  I think we should inquire about this to Discord, especially since I could see people adding "yesno" or other variations to this.

Reply:
Yeah, that's fair - I'll reach out to them and ask for clarification.

Reply:
Updated the rule to check for a couple different variations based on their feedback.